ts/components/wallet/payment/PickPaymentMethodBaseListItem.tsx

```diff
@@ -69,12 +67,12 @@ const PickPaymentMethodBaseListItem: React.FC<Props> = ({
         />
         <VSpacer size={16} />
         <View style={styles.paymentMethodInfo}>
-          <H4 weight={"Semibold"} color={"bluegreyDark"} numberOfLines={1}>
+          <H6 color={"bluegreyDark"} numberOfLines={1}>
```

Comment:
In that case, we have H4 and H5 that became H6, which appears to be an error but we can ignore these changes as this PR https://github.com/pagopa/io-app/pull/6210 will remove the file.